symengine/functions.h

```diff
@@ -252,6 +252,13 @@ class TrigFunction : public OneArgFunction
     TrigFunction(RCP<const Basic> arg) : OneArgFunction(arg){};
 };
 
+class InverseTrigFunction : public OneArgFunction
```

Comment:
Can you make both of these types inherit from a base class like @srajangarg said?